tests/transformers/searchservice/createDirectAnswer.ts

```diff
@@ -59,4 +59,26 @@ it('can create a FieldValueDirectAnswer', () => {
     fieldType: 'phone'
   };
   expect(actualDirectAnswer).toMatchObject(expectedDirectAnswer);
+});
+
+it('can create a custom FieldValueDirectAnswer', () => {
+  const apiFieldValueDirectAnswer = {
+    type: 'FIELD_VALUE',
+    answer: {
+      value: {
+        color: 'yellow',
+        os: 'android'
+      },
+      entityName: 'Obama Phone',
+      fieldName: 'Phone Description',
+      fieldApiName: 'phoneDescription',
+      fieldType: 'c010101.specialtype'
```

Comment:
this is a super nit but I don't think a businessId will ever start with a 0? (could be wrong) maybe we can use our test account's businessId 3350634

Reply:
I mean it's kind of exposing stuff in a public repo, but I don't really care too much about that personally...? some other semi-realistic business id also works for me

Reply:
sure thing